enzyme/Enzyme/TypeAnalysis/TypeAnalysisPrinter.cpp

```diff
@@ -64,10 +64,12 @@ using namespace llvm;
 #endif
 #define DEBUG_TYPE "type-analysis-results"
 
+extern "C" {
 /// Function ActivityAnalysis will be starting its run from
 llvm::cl::opt<std::string>
     FunctionToAnalyze("type-analysis-func", cl::init(""), cl::Hidden,
```

Comment:
can you change the name of this, perhaps EnzymeFunctionToAnalyze, if it is expected to be exported now